src/halmos/cheatcodes.py

```diff
@@ -643,9 +681,10 @@ def handle(sevm, ex, arg: ByteVec, stack, step_id) -> ByteVec | None:
 
         elif funsig == hevm_cheat_code.label_sig:
             addr = extract_bytes(arg, 4, 32)
-            label = extract_string_argument(arg, 1)
 
             # TODO: no-op for now
+            # label = extract_string_argument(arg, 1)
+
```

Comment:
does ruff do the deadcode elimination as well? that'd be quite amazing but also sound scary.

Reply:
it did point out that we defined `label` but never used it -- it doesn't automatically eliminate it